whiterabbit/src/main/java/org/ohdsi/whiteRabbit/scan/SourceDataScan.java

```diff
@@ -419,6 +449,8 @@ public void processValue(String value) {
 				emptyCount++;
 
 			if (!isFreeText) {
+				if (!valueCounts.contains(value))

```

Comment:
Instead of testing contains, you can use the return value of `boolean newlyAdded = valueCounts.add(value)`